SingularityService/src/main/java/com/hubspot/singularity/resources/DeployResource.java

```diff
@@ -134,8 +138,13 @@ public SingularityRequestParent deploy(@ApiParam(required=true) SingularityDeplo
 
     SingularityPendingDeploy pendingDeployObj = new SingularityPendingDeploy(deployMarker, Optional.<SingularityLoadBalancerUpdate> absent(), DeployState.WAITING, deployProgress, updatedValidatedRequest);
 
-    checkConflict(deployManager.createPendingDeploy(pendingDeployObj) != SingularityCreateResult.EXISTED,
-        "Pending deploy already in progress for %s - cancel it or wait for it to complete (%s)", requestId, deployManager.getPendingDeploy(requestId).orNull());
+    boolean deployExists = deployManager.createPendingDeploy(pendingDeployObj) == SingularityCreateResult.EXISTED;
+    if (deployExists && forceDeploy.or(false)) {
+      cancelDeploy(requestId, deployManager.getInUseDeployId(requestId).orNull());
```

Comment:
Unsure if there is a restart method or if I need to do something after hitting the cancel